src/convert.ts

```diff
@@ -2004,6 +2004,20 @@ export default function convert(config: ConvertConfig): ESTreeNode | null {
       });
       break;
 
+    case SyntaxKind.BigIntLiteral: {
+      const raw = ast.text.slice(
+        (result as any).range[0],
+        (result as any).range[1]
+      );
+      const value = raw.slice(0, -1); // remove suffix `n`
+      Object.assign(result, {
+        type: AST_NODE_TYPES.BigIntLiteral,
+        raw,
+        value
+      });
```

Comment:
I'm still not sure what should this node look like. `@babel/parser` parsed them as:

```ts
interface BigIntLiteral {
  type: "BigIntLiteral";
  value: string;
  extra: {
    raw: string;
    rawValue: string;
  }
}
```

but the alignment test rejected the `extra` field and expected:

```ts
interface BigIntLiteral {
  type: "BigIntLiteral";
  value: string;
  raw: string;
}
```

which is the current implementation.

Reply:
Hmm yeah seems there isn't quite a consensus yet: https://github.com/estree/estree/issues/169

There hasn't been any update on that issue for a while...

I think the least likely of all is having that extra object involved in ESTree, so maybe we should just ignore the AST comparison for now?

Reply:
`extra` is already ignored, I just want to make sure if it's fine to output the node that's different from babel's. 😅

https://github.com/JamesHenry/typescript-estree/blob/8e7037599248b77d725ca53317231417f792f43e/tests/ast-alignment/utils.ts#L102-L105

Reply:
Yeah I think it's fine, I think we just have to accept that as experimental things get added they may be subject to change.

We can always publish new major versions in future if we need to amend it